src/Common/CodeCracker.Common/Extensions/Extensions.cs

```diff
@@ -30,5 +30,92 @@ public static string ToLowerCaseFirstLetter(this string text)
             if (text.Length == 1) return text.ToLower();
             return char.ToLowerInvariant(text[0]) + text.Substring(1);
         }
+
+        public static bool IsCSharpKeyword(this string name)
+        {
+            switch (name)
```

Comment:
As this will be invoked very frequently, create a static array with the strings and check it with `Array.IndexOf(keywords, "aa") > -1`